defaults/gce_config.yaml

```diff
@@ -5,7 +5,7 @@ gce_project: '' # empty mean default one, can be overwritten to use different on
 
 gce_image_db: '' # so we can use `scylla_version` as needed
 gce_image_loader: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7'
-gce_image_monitor: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7'
+gce_image_monitor: 'https://www.googleapis.com/compute/alpha/projects/skilled-adapter-452/global/images/scylla-monitoring-cloud-ubuntu-4-0-2-2022-09-19t14-39-35z'
```

Comment:
should try it with v1 as well (even that this is the url google is returning for this image)